### PR TITLE
Fix scale text

### PR DESCRIPTION
**Pull request by dstrawberrygirl**
Addresses feedback from PR, adds a line break to the scale text, increases the font size, and recalculates the scale on initial load and on window resize (in addition to on zoom).
[171616026]

### CODE REVIEW

File: js/components/seismic-eruptions-map.js [hunk 1]
```diff
@@ -258,7 +263,14 @@ export default class SeismicEruptionsMap extends PureComponent {
           }
         </Map>
         {!config.showUserInterface &&
-          <div className='scale-markers'>{zoom > 3 ? `Scale: ${scaleWidth}km x ${scaleHeight}km   Zoom level: ${zoom}` : ''}</div>
+          <div className='scale-markers'>
+            {zoom > 3 &&
```
Comment (pjanik) [inline js/components/seismic-eruptions-map.js:267]: Why do we have this check and why this particular value (3)?

**Comment by pjanik** [comment]
Looks good. One inline question about the magic number. ;)

File: js/components/seismic-eruptions-map.js (hunk 1 above)
Comment (dstrawberrygirl) [reply js/components/seismic-eruptions-map.js:267]: Leaflet api claims that calculating distances will return meters, but when the distances are significantly large (somewhere over 100k km) it seems to return values in km (I tested with logging the raw values before applying the various m => km, rounding, and `toLocaleString`). At zoom level 3 it was hit or miss as to whether the numbers appeared within a reasonable range, and it depended heavily on screen size. Added to the fact that map projection has role to play that becomes particularly noticeable at lower levels of zoom, I took the simplest approach of sticking to 4 and above.

**Comment by dstrawberrygirl** [comment]
The width and height calculations are currently not needed for the class tests, but have left the code commented out in case we have a last minute change request. I also left a note about the obscure zoom behavior for future.

**Pull request merged**